scripts/dcp1-version/populate_dcp_version.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import os
+import uuid
+from datetime import datetime
+
+from bson import JAVA_LEGACY, CodecOptions
+from pymongo import MongoClient
+
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+DCP1_GS_FILES_LS = os.environ.get('DCP1_GS_FILES_LS')
+DB_HOST = os.environ.get('DB_HOST', 'localhost')
+DB_PORT = os.environ.get('DB_PORT', 27017)
+
+DB_CLIENT = MongoClient(DB_HOST, 27017)
+DB = DB_CLIENT.get_database('admin')
+
+DCP_VERSION_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+
+COLLECTION_MAP = {
+    'project': 'project',
+    'cell_line': 'biomaterial',
+    'cell_suspension': 'biomaterial',
+    'collection_protocol': 'protocol',
+    'differentiation_protocol': 'protocol',
+    'dissociation_protocol': 'protocol',
+    'donor_organism': 'biomaterial',
+    'enrichment_protocol': 'protocol',
+    'ipsc_induction_protocol': 'protocol',
+    'library_preparation_protocol': 'protocol',
+    'organoid': 'biomaterial',
+    'process': 'process',
+    'sequence_file': 'file',
+    'sequencing_protocol': 'protocol',
+    'specimen_from_organism': 'biomaterial',
+    'supplementary_file': 'file'
+}
+
+
+def load_lines_from_file(file: str):
+    with open(file) as f:
+        data = [line.rstrip() for line in f]
+        return data
+
+
+def find_type_uuid_version(line: str):
```

Comment:
Is there an example of the line format you expect here?

Reply:
it's in the test